.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 🤔 Feature Requests & Questions 
+    url: https://github.com/thomasheyenbrock/remix-graphql/discussions
+    about: Please ask and answer questions here.
+  - name: 💬 Remix Discord Channel 
+    url: https://rmx.as/discord
+    about: Interact with other people using Remix 📀
+  - name: 💬 New Updates (Twitter)
+    url: https://twitter.com/remix_run
+    about: Stay up to date with Remix news on twitter
+  - name: 🍿 Remix YouTube Channel
+    url: https://www.youtube.com/channel/UC_9cztXyAZCli9Cky6NWWwQ
+    about: Are you a techlead or wanting to learn more about Remix in depth? Checkout the Remix YouTube Channel 
```

Comment:
I'm not sure if I'd wanna redirect users who see an issue with this specific lib to places that are about Remix in general 🤔 But I also didn't use these issue templates before. Would you consider this to be "best practice"? Do you have examples of other libraries that do sth similar?